src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -176,7 +176,7 @@ public function searchBarAction(Request $request): Response
             $this->cspHandler->disableCsp();
         }
 
-        if (!$request->hasSession()) {
+        if (true === $request->attributes->getBoolean('_stateless') || !$request->hasSession()) {
```

Comment:
`true === ` should be removed. There is no need to turn a boolean into another (same) boolean